src/batArray.mlv

```diff
@@ -822,6 +822,22 @@ let insert xs x i =
      with Invalid_argument _ -> true
 *)
 
+let remove_at i src =
+  let x = src.(i) in (* keep the bound check in there *)
+  let n = length src in
+  let dst = make (n - 1) x in
```

Comment:
For the record: In theory, Array.init would permit to initialize `dst` only once taher than twice (first with `x`, second with actual values). In practice IIRC it's the same under the hood.

Reply:
I thought about this.
If it's faster, why not.
I think it might also make the code more readable.
Thanks for the review by the way.